### DISCUSSION TITLE
Serving static files for Frontend

### DISCUSSION THREAD

**Discussion by Letscat** [go-fuego/fuego#107] [Q&A]
Hey there, i am currently working on a webapp with go, and was wondering if it is possible to serve a html frontend not using rendered templates but static content.

the goal here is to have all my endpoints that return json under an "/api/endpoint" prefix.
And serving a react frontend under "/"

Using gin this would look something like this:
  router.Use(static.Serve("/", static.LocalFile("./views", true)))
  
  
It could be that this sample implements something like this, but im not exactly sure how it is done and how it works
https://github.com/go-fuego/fuego/blob/main/examples/full-app-gourmet/server/server.go

**Comment by EwenQuim** [comment]
Yes it's exactly like you see on the full-app-gourmet example !

I'll make a dedicated tutorial about it soon.
